xtask/src/main.rs

```diff
@@ -46,10 +46,11 @@ USAGE:
     cargo xtask install [FLAGS]
 
 FLAGS:
-        --client-code    Install only VS Code plugin
-        --server         Install only the language server
-        --mimalloc       Use mimalloc for server
-    -h, --help           Prints help information
+        --client-code[=CLIENT]    Install only VS Code plugin.
```

Comment:
```suggestion
        --client [=CLIENT]    Install only VS Code plugin.
```

we never had any other clients, so it's better to simplify this while we are at it.